### DISCUSSION TITLE
unable to push due to different user access denied

### DISCUSSION THREAD

**Discussion by Rajyavardhan-singh** [community/community#51201] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

i have two github accounts one created recently.
i have updated the user name and email in the config file for the new account but while running push command it is giving me an error

]remote: Permission to Rajyavardhan-singh/folder.git denied to rajyavardhansr.

where i have to make changes so that i can push code to new account.

**Comment by matej-kaska** [comment]
Try to look in command "git config --list" if there is "credential.helper=manager" you need to change credentials in OS.

https://stackoverflow.com/questions/22844806/how-to-change-my-git-username-in-terminal

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Rajyavardhan-singh ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
